duckduckgo/Android
# ADS: Unprotected Sites migration

Task/Issue URL: https://app.asana.com/0/0/1203467500325941/f

### Description

- Migrate Unprotected Sites screen to new components 
- Created new CustomAlertDialog component

### Steps to test this PR

- Install from this branch
- Go to Settings > Unprotected Sites
- [ ] Check screen look as expected

### UI changes
| Before  | After |
| ------ | ----- |
![Screenshot_20221206_180155_DuckDuckGo](https://user-images.githubusercontent.com/20798495/205987523-0e85ae6f-3fb7-4e9a-a71c-23f396584966.jpg)|![Screenshot_20221206_141121_DuckDuckGo](https://user-images.githubusercontent.com/20798495/205987409-c1bfbb9e-2ef6-439a-95e2-87bd28ba24e8.jpg)|
